guides/source/testing.md

```diff
@@ -37,10 +37,12 @@ controllers/    helpers/        mailers/        test_helper.rb
 fixtures/       integration/    models/
 ```
 
-The `models` directory is meant to hold tests for your models, the `controllers` directory is meant to hold tests for your controllers and the `integration` directory is meant to hold tests that involve any number of controllers interacting. There is also a directory for testing your mailers and one for testing view helpers.
+The `models` directory is meant to hold tests for your models, the `controllers` directory is meant to hold tests for your controllers, routes and views and the `integration` directory is meant to hold tests that involve any number of controllers interacting. There is also a directory for testing your mailers and one for testing view helpers.
```

Comment:
We prefer the oxford comma in documentation so change "tests for your controllers, routes and views" to "tests for your controllers, routes, and views" to have the comma after routes.

Additionally this change has created a bit of a run-on sentence. Can you change the first sentence to be 2 sentences, like so:

> The `models` directory is meant to hold tests for your models, the `controllers` directory is meant to hold tests for your controllers, routes, and views. The `integration` directory is meant to hold tests that involve any number of controllers interacting